baseplate/sidecars/event_publisher.py

```diff
@@ -203,11 +230,26 @@ def publish_events() -> None:
 
     metrics_client = metrics_client_from_config(raw_config)
 
-    event_queue = MessageQueue(
-        "/events-" + args.queue_name,
-        max_messages=cfg.max_queue_size,
-        max_message_size=MAX_EVENT_SIZE,
-    )
+    if args.use_in_memory_queue:
+        # start a thrift server that will be used to communicate with the main baseplate app
+        # this should not happen here, where are sidecars running?
+        processor = RemoteMessageQueueService.processor(RemoteMessageQueueHandler())
+        transport = TSocket.TServerSocket(host='127.0.0.1', port=9090)
```

Comment:
This port number should proably use a constant that is shared with the client code, to remove the chance of them using different values.